nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyArrayTests.java

```diff
@@ -0,0 +1,91 @@
+package com.uber.nullaway;
+
+import com.google.errorprone.CompilationTestHelper;
+import java.util.Arrays;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class NullAwayJSpecifyArrayTests extends NullAwayTestsBase {
+
+  @Test
+  public void ArrayTypeAnnotationDereference() {
```

Comment:
(for all methods in this class) Java method names should be lower Camel Case https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names